doc/epub.md

```diff
@@ -156,6 +158,9 @@ of the image,  increasing accessibility for blind users.
 
 [KindlePreviewer]: https://www.amazon.com/Kindle-Previewer/b?node=21381691011
 [KindleGen]: https://www.amazon.com/gp/feature.html?docId=1000765211
+[Linux]: https://archive.org/details/kindlegen2.9
+[Mac OSX]: https://web.archive.org/web/20190905040839/https://www.amazon.com/gp/feature.html?ie=UTF8&docId=1000765211
+[Windows]: https://archive.org/details/kindlegen_win32_v2_9
```

Comment:
I think it's confusing to make the reference link `[Linux]` refer to this rather than to something about Linux.
Better to make the reference `[KindleGenLinux]` (or something like that), you can then form the link as `[Linux][KindleGenLinux]`.

Reply:
Thanks. Done.